docs/core/stories.rst

```diff
@@ -86,15 +157,14 @@ contains checkpoints:
 
```

Comment:
Can you change the above to:

```
    ## first story
    * greet
       - action_ask_user_question
    > check_asked_question

    ## user affirms question
    > check_asked_question
    * affirm
      - action_handle_affirmation
    > check_handled_affirmation

    ## user denies question
    > check_asked_question
    * deny
      - action_handle_denial
    > check_handled_denial

    ## user leaves
    > check_handled_denial
    > check_handled_affirmation
    * goodbye
      - utter_goodbye
```

I realised we never mention that you can use multiple checkpoints

Reply:
Never knew you could do that!